commons-app/src/main/java/cz/cuni/mff/xrg/odcs/commons/app/user/User.java

```diff
@@ -81,6 +94,12 @@
     @OneToOne(mappedBy = "user", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
     private UserNotificationRecord notification;
 
+    @Column(table = "usr_extuser", name = "id_extuser")
+    private String externalIdentifier;
+
+    @Transient
+    private Organization organization;
```

Comment:
This is a hack, see 
https://github.com/UnifiedViews/Core/issues/341, organization should normally persist to DB.